lib/src/google_api_availability.dart

```diff
@@ -12,14 +10,15 @@ import 'models/google_play_services_availability.dart';
 class GoogleApiAvailability {
   const GoogleApiAvailability._();
 
+  ///Only visible for testing purposes
```

Comment:
nit: let's describe this a little bit more detailed. Something like:

```suggestion
  /// Creates an instance of the [GoogleApiAvailability] class. This 
  /// constructor is exposed for testing purposes only and should not be used 
  /// by clients of the plugin as it may break or change at any time. 
```